tests/bootstrap-api-coverage.php

```diff
@@ -1,7 +1,11 @@
 <?php
+
 /**
- * @copyright 2015-2016 Contentful GmbH
+ * This file is part of the contentful.php package.
```

Comment:
Personally I'd rather see the big change then this. Otherwise this repo will be inconsistent for years...